docs/content/configuration/csi_ug_config_create_vol_snapshotclass.md

```diff
@@ -5,30 +5,35 @@ Create a VolumeSnapshotClass YAML file to enable creation and deletion of volume
 **Note:**
 
 -   IBM® FlashCopy® function is referred to as the more generic volume snapshots and cloning within this documentation set. Not all supported products use the FlashCopy function terminology.
--   For volume snapshot support, the minimum orchestration platform version requirements are Red Hat® OpenShift® 4.4 and Kubernetes 1.17.
 
-In order to enable creation and deletion of volume snapshots for your storage system, create a VolumeSnapshotClass YAML file, similar to the following demo-snapshotclass.yaml.
+In order to enable creation and deletion of volume snapshots for your storage system, create a VolumeSnapshotClass YAML file, similar to the following `demo-snapshotclass.yaml`.
```

Comment:
@ArbelNathan [do we want](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r704256279) to update the filenames (wherever applicable)? e.g.
```suggestion
In order to enable creation and deletion of volume snapshots for your storage system, create a VolumeSnapshotClass YAML file, similar to the following `demo-volumesnapshotclass-config-secret.yaml`.
```

Reply:
yes, sure

Reply:
DONE

Reply:
@rivkap sorry for the confusion, but my suggestion here is no longer relevant, since the topology is now in a separate topic

Reply:
@oriyarde can you point out where it should be removed / replaced back with demo-snapshot?
should this end up just waiting for the final yaml examples to be copied in?
updated were made in the following files:
1. csi_ug_config_create_snapshots.md
2. csi_ug_config_create_vol_snapshotclass.md 
3. csi_ug_config_create_vol_snapshotclass_topology.md

Reply:
1. and 2. should not contain `config-secret`, since it is covered in 3.

Reply:
DONE